spec/system/task_spec.rb

```diff
@@ -1,36 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let!(:project) { create(:project) }
+  let(:task) { create(:task, project_id: project.id) }
+  let(:completed_task) { create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday) }
```

Comment:
定義したtraitを呼び出す形にすると、以下のように記述を減らすことができますね。
```suggestion
  let(:completed_task) { create(:task, :completed) }
```